lib/helper.js

```diff
@@ -135,6 +135,23 @@ PokerHelper.prototype.nextActivePlayer = function(lineup, playerPos) {
   return newPos;
 }
 
+PokerHelper.prototype.findMaxRaise = function(lineup, dealer, lastRoundMaxBet) { 
```

Comment:
i would either pass in the BB as param, and return it, or return -1, saying that it is not this functions scope to determine this case.